packages/client-core/src/components/Glass/ChatMenu.tsx

```diff
@@ -181,9 +181,14 @@ export const ChatMenu = () => {
         })
 
         return isOwnGroup || isNotification ? (
-          groupedMessage
+          <div
+            key={groupIndex}
+            className={twMerge('mb-6 flex w-full flex-col gap-y-2', isOwnGroup ? 'items-end' : 'items-center')}
+          >
+            {groupedMessage}
+          </div>
         ) : (
-          <OtherChat key={groupIndex}>
+          <OtherChat key={groupIndex} className="mb-6">
```

Comment:
🐑 `OtherChat` doesnt need to be extended like this since its only used here, so this `mb-6` could be moved up to line 90 with the other styles